.gitignore

```diff
@@ -1,3 +1,4 @@
 vendor/
-composer.lock
 build/
+/nbproject/
+composer.lock
```

Comment:
@J-Mose may have final word on this, but community practice is to include `compose.lock` as a checked in file.